README.md

```diff
@@ -125,6 +131,7 @@ prebuild [options]
   --include-regex               (regex to match files that will be distributed [default: '\.node$'])
   --libc                        (use provided libc rather than system default)
   --backend                     (specify build backend, default is 'node-gyp')
+  --format                      (specify additional parameters for `node-gyp` backend)
```

Comment:
This description isn't right (it's not about multiple parameters), but let's tweak that later